ale_linters/rust/cargo.vim

```diff
@@ -11,10 +13,18 @@ function! ale_linters#rust#cargo#GetCargoExecutable(bufnr) abort
     endif
 endfunction
 
+function! ale_linters#rust#cargo#GetCommand(buffer) abort
+    let l:command = g:ale_rust_cargo_use_check
+    \   ? 'check'
```

Comment:
I follow Donald Knuth's style for binary operators across many lines, and place the two ternary symbols on the left also.